modules/apps/frontend-taglib/frontend-taglib-sample-web/bnd.bnd

```diff
@@ -0,0 +1,5 @@
+Bundle-Name: Liferay Frontend Taglib Sample Web
+Bundle-SymbolicName: com.liferay.frontend.taglib.sample.web
+Bundle-Version: 1.0.0
+Export-Package: com.liferay.frontend.taglib.sample.web.constants
```

Comment:
Why do we need to export constants package? A sample module should normally not export anything.

Reply:
Done!